languages/rust/exercises/concept/structs/.docs/hints.md

```diff
@@ -0,0 +1,16 @@
+## General
```

Comment:
Currently, there are only general hints in this file. For most exercises, we also have task-specific hints, to help people with solving a particular tasks. An example of this is [this hints.md file](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/floating-point-numbers/.docs/hints.md), which has both a general hints, but also task-specific hints.

For more information on this, see [the documentation](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd).